src/improvements/tasks/sep/009-opcm-upgrade-prestate-v300-op-ink/VALIDATION.md

```diff
@@ -0,0 +1,244 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the upgrade transaction which you are
+signing.
+
+The steps are:
+
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the transaction input](#understanding-task-calldata)
+3. [Verifying the state changes](#state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and
+the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Optimism Foundation
+>
+> - Domain Hash: `0x37e1f5dd3b92a004a23589b741196c8a214629d4ea3a690ec8e41ae45c689cbb`
+> - Message Hash: `0xfb07e7203c346d15eef032fd4f2a701bb8b6d238ddf1bc039e6948679bfbe244`
+>
+> ### Security Council
+>
+> - Domain Hash: `0xbe081970e9fc104bd1ea27e375cd21ec7bb1eec56bfe43347c3e36c5d27b8533`
+> - Message Hash: `0x6b925fb3038474120f3e0ea58b70c686ce8911846205441dd0d0466d4bab2479`
+
+## Understanding Task Calldata
+
+This document provides a detailed analysis of the final calldata executed on-chain for the OPCM updatePrestate() function.
+
+By reconstructing the calldata, we can confirm that the execution precisely implements the approved upgrade plan with no unexpected modifications or side effects.
+
+### Inputs to `opcm.updatePrestate()`
+
+For each chain being updated, the `opcm.updatePrestate()` function is called with a tuple of two elements:
+
+1. OP Sepolia Testnet:
+  - SystemConfigProxy: [0x034edD2A225f7f429A63E0f1D2084B9E0A93b538](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/sepolia/op.toml#L58)
+  - ProxyAdmin: [0x189aBAAaa82DfC015A588A7dbaD6F13b1D3485Bc](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/sepolia/op.toml#L59)
+  - AbsolutePrestate: [0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6](https://www.notion.so/oplabs/Isthmus-Sepolia-Mainnet-1d2f153ee162800880abe1b47910c071)
```

Comment:
I like the content of this notion link, should it be public?

Reply:
I'm not sure. I was advised by Seb to link this. I think the doc is private.

Reply:
I think it makes sense to make these prestate reviews public 👍🏻